tests/Controller/Kobo/AbstractKoboControllerTest.php

```diff
@@ -119,4 +122,22 @@ protected function getMockClient(string $returnValue): ClientInterface
         $handlerStack = HandlerStack::create($mock);
         return new Client(['handler' => $handlerStack]);
     }
+
+    /**
+     * Return the number of doctrine queries
+     * Make sure to call "$client->enableProfiler();" before making the request
+     */
+    protected function getNumberOfQueries(KernelBrowser $client): int{
+        $profile = $client->getProfile();
```

Comment:
I don't like that tests get dependent on a package that should only be used for dev... IMHO it was wrong in the beginning to allow the profilerbundle in test env